src/Button/Button.mdx

```diff
@@ -83,16 +118,22 @@ Use the `size` prop to change the size of your button.
 Add the `disabled` prop for disabled button state style.
 
 <Playground>
-  <Button disabled>I'm disabled</Button>
+  <Flex justifyContent="space-around">
+    <Button disabled>I'm Disabled</Button>
+    <Button variant="secondary" disabled>I'm Disabled</Button>
+    <Button variant="grey" disabled>I'm Disabled</Button>
+    <Button variant="success" disabled>I'm Disabled</Button>
+    <Button variant="danger" disabled>I'm Disabled</Button>
+  </Flex>
 </Playground>
 
 ### Loading
 
 Add the `loading` prop for loading button state style.
 
 <Playground>
-  <Button.Group>
+  <Flex justifyContent="space-around">
     <Button loading>I'm loading</Button>
     <Button loading outline>I'm loading</Button>
-  </Button.Group>
+  </Flex>
```

Comment:
Why were button groups replaced with flex? Should Button.Group be updated to accommodate this use case, if it is common?